src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { Input } from 'reactstrap'
+import './ProfileModifier.css'
+
+export default function ProfileCard (props) {
+  const [month] = useState(parseInt(new Date().getMonth()) + 1)
+  const [pic] = useState(getPic())
+
+  function getPic () {
+    let pic
+    if (month === 3 || month === 4 || month === 5) { pic = require('./Image/spring.jpg') } else if (month === 6 || month === 7 || month === 8) { pic = require('./Image/summer.jpeg') } else if (month === 9 || month === 10 || month === 11) { pic = require('./Image/fall.jpeg') } else if (month === 12 || month === 1 || month === 2) { pic = require('./Image/winter.jpeg') }
+    return pic
+  }
+
+  return (
+    <div id='enclose'>
+      <img id='clip' alt='side' src={pic} />
+      <img id='clip2' alt='side' src={pic} />
+      <div id='profileBox'>
+        {props.fields.map((field, ind) => (
+          <h3 key={ind} id='innerTextTop'>
+            <b>{field.front}:</b> {field.end}
+          </h3>
+        ))}
+        <h3 id='innerTextTop'>
+          New Password: <Input type='password' />
+        </h3>
+        <h3 id='innerTextTop'>
+          Confirm Password: <Input type='password' />
+          <button style={{ marginTop: '10px' }}>Change Password</button>
```

Comment:
This button should be disabled until the `password` and `confirmedPassword` hooks are equal.